### PR TITLE
feat: add see all/less toggle to NFTs sections

### LINKED ISSUE #219: [Task]: Add see all/less toggle to NFT section
Follow [design](https://www.figma.com/file/Fe2hyux4KGmKjeni2nb2mg/Sugar-Rush?node-id=0%3A1) to add `See all` and `See less` buttons. 
By default, show only as many nfts as they fit in one row.
Then, upon click on `See all`, show all NFTs.
Then, upon click on `See less`, show back again only one row.

image
image
image

### PR DESCRIPTION
**Pull request by aspnxdd**
Fixes #219 

## Changes

  - Adds to toggle buttons to show all/less for both Minted and Unminted NFT lists.
  - Changes the minimum amount of NFTs to be 5 (to fit in 1 row in mostly all types of screens).
  - Changes the key identifier from the index of the arrays looped to a unique item from those arrays, specifiaclly the mintAddress to base58.
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

**Review by agus-xyz — COMMENTED** [review]
ou yeah, really nice @aspnxdd. I was testing a bit the new functionality and I observed the following...

The `Show all` button is displayed always (even if there is no more items to show). It should only appear if there are more NFTs available than the ones displayed 
image

Then on the other hand, in case it is not possible to mint further NFTs (and the MINT card is not shown), then the number of items in the 1st row is actually 6, not 5... 
image

File: contexts/ArrayWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+type FilterArray = [Array<any>, boolean]
+
+export const FilterArrayContext = createContext<FilterArray>([[], false])
+
+enum State {
+    ShowAll = 'Show All',
+    ShowLess = 'Show Less',
+}
+
+export const ArrayWrapper: FC<{ array: Array<any>; children: ReactNode; limit: number }> = ({
```
Comment (begonaalvarezd) [inline contexts/ArrayWrapper.tsx:12]:
```suggestion
export const ArrayWrapper: FC<{ array: Array<any>; children: ReactNode; minimum: number }> = ({
```
`limit` --> `minimum` I feel is more clear

**Review by begonaalvarezd — APPROVED** [review]

**Review by begonaalvarezd — APPROVED** [review]
> Then on the other hand, in case it is not possible to mint further NFTs (and the MINT card is not shown), then the number of items in the 1st row is actually 6, not 5...

this comment from Agus is still to be fixed 🙏🏼

**Review by begonaalvarezd — APPROVED** [review]
there we go 🚀

**Pull request merged**